qcdb/intf_nwchem/harvester.py

```diff
@@ -542,6 +539,23 @@ def format_modelchem_for_nwchem(name, dertype, ropts, sysinfo, verbose=1):
             ropts.require('NWCHEM', 'tce__eaccsd', True, **kwgs)
             ropts.suggest('NWCHEM', 'tce__nroots', 4, **kwgs)
 
+    elif lowername == 'nwc-sodft':
+        ropts.suggest('NWCHEM', 'xc', 'b3lyp', **kwgs)
+        mdccmd = f'task sodft {runtyp} \n\n'
+
+    #DFT- relativistic
+    elif lowername == 'nwc-zora':
```

Comment:
"name" here is exclusively "method" (which is why we needed to find another way other than "nwc-tce-ccsd" for the tce/cc selection), and the relativistic corrections can be used with many methods, need to save this spot for the method, not the relativistic approach. (I checked with the manual "Since these approximations only modify the integrals, they can in principle be used at all levels of theory."). so these `elif`s should go away, I think. users should `set_options({'nwchem_relativistic__zora': True})`, then `qcdb.energy('nwc-b3lyp')` to get the same result.

Reply:
I see for relativistic. @loriab, for the tce-cc naming, is the current elif structure alright with the `nwc-{method}` and specifying necessary `tce` from qc module? I don't believe any of the `nwc` calls have been two-tiered other than something like `nwc-eom-ccsd`

Reply:
the "nwc-tce-ccsd" was an early iteration. the current structure is ok for now but one must deliberately set "tce" for rohf, for example. will rework it to be more like https://github.com/psi4/psi4/blob/master/psi4/driver/procrouting/proc.py#L75 later.